### PR TITLE
JS Bitwise Operators: Documentation

### PR DESCRIPTION
**Pull request by prishit55**
<hr>

## Description 📜
I've made a new file in "JavaScript" called "JavaScript Bitwise Operators", and under that file a md file called "Js Bitwise" which is a brief discussion about various bitwise operators in Js has been created. #7310 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)
<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/Javascript/JavaScript Bitwise Operator/JS Bitwise.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```
+## 5. Left Shift
+Left shift operator shifts the bits of the first operand to the left, and second operand describes the positions to be shifted.
+### Example
+```javascript
+let a=5;            //bit representation: 00000000000000000000000000000101
+let b=1;           //bit representation: 00000000000000000000000000000001
+console.log(a<<b); //output 00000000000000000000000000001010 (that is 10)
+``` 
+### 6. Right Shift
```
Comment (kashishahuja2002) [inline Web_Development/Javascript/JavaScript Bitwise Operator/JS Bitwise.md:73]: Wrong level of heading, use ##

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Good work.

File: Web_Development/Javascript/JavaScript Bitwise Operator/JS Bitwise.md (hunk 1 above)
Comment (prishit55) [reply Web_Development/Javascript/JavaScript Bitwise Operator/JS Bitwise.md]: Done !!

**Review by kashishahuja2002 — APPROVED** [review]
LGTM. Tag the supervisor of your batch for further reviewing and merging the PR.

**Pull request merged**